configs/mmdet3d/mono-detection/mono-detection_openvino_dynamic-384x1280.py

```diff
@@ -0,0 +1,6 @@
+_base_ = ['./mono-detection_dynamic.py', '../../_base_/backends/openvino.py']
+
```

Comment:
have you tested on openvino backend?

Reply:
Actually, not yet, because jetson not support openvino.
I suppose regression would help run the openvino part.

Reply:
I tested openvino on cpu, found following error:
OpenVINO does not support the following ONNX operations: mmdeploy.MMCVModulatedDeformConv2d

I checked mmdeploy doc, seems the op not supported in OpenVINO.
How to do with this?

Reply:
hi, we can skip openvnino for now. This custom op may not be included in openvino operator sets.

Reply:
Done.